### ISSUE TITLE
Editor not showing mouse pointer.

### ISSUE THREAD

**Issue by NemesisMate** [Simsilica/SimArboreal-Editor#1]
The last editor release is opening fine under linux but there is no mouse pointer so it can't be used :S.

**Comment by pspeed42** [comment]
Weird... can you try pressing the space bar?

**Comment by NemesisMate** [comment]
-O-, it works!. I tried "all" the keys but that one :D. If y press the space bar it toggles the visibility of the cursor. It is made intentionally?, if so, what about a hint in anyplace of the screen saying it?.

**Comment by pspeed42** [comment]
Yeah, there should be a hint probably.  All of those demos work that way.
Even the IsoSurfaceDemo uses space to toggle between mouse and movement.
Old habit and I don't even think about it anymore. :)

Probably when in 'fly mode' I should put a label at the bottom of the
screen "Press Space To Show Mouse" and "Press Space to Move Camera" when in
UI mode or something.  I'll have to think about it.

Also, I think it's a bug that SimArboreal doesn't start out in UI mode...
may be a regression.

**Comment by NemesisMate** [comment]
Yeah, maybe if it starts with the cursor is quite obvious to press the space bar to toggle it. Maybe is also obvious if not but I tried to press may buttons yesterday and I directly thought it wasn't possible xD (maybe a little bit dumb xD) but there's always people worse!.
A fast "fix" can be put that in the repo readme (while it isn't really fixed) as there was the first place I looked for (well, the second, the first was the packed changelog -O-).
